pkg/cmd/server/admission/init.go

```diff
@@ -22,6 +23,7 @@ type PluginInitializer struct {
 	RESTClientConfig      restclient.Config
 	Informers             shared.InformerFactory
 	ClusterQuotaMapper    clusterquotamapping.ClusterQuotaMapper
+	DefaultRegistryFn     imageapi.DefaultRegistryFunc
```

Comment:
> Master config should eventually allow alternate names for the registry, and it'll be an array of strings. Kubelets will not be allowed to have different decisions for what an image registry is.

And you're saying, "not in this pull"?  I can accept hat, but it's a significant limitation.